app/admin/documents.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register Document do
+  permit_params :published, :file, :category
+   actions :all, except: [:show]
+
+  menu if: proc { authorized? :index, %i[active_admin Productor] } # display menu according to ActiveAdmin::Policy
```

Comment:
The `Productor` policy manages the `Document` admin resource?

Reply:
no, is an error.